packages/backend-core/src/features/index.ts

```diff
@@ -150,6 +156,26 @@ export class FlagSet<V extends Flag<any>, T extends { [key: string]: V }> {
         }
       }
 
+      const license = ctx?.user?.license
+      if (license) {
+        for (const feature of license.features) {
+          const flag = this.flags[feature]
```

Comment:
NAB: As far as I can see, this `flag` var is not used anywhere else, therefore it could be simplified to indicate that it's not used anywhere else:
`if (!this.flags[feature]) continue`

Another point is that we have a readonly property called `flags` as well as the locally scoped `flags` variable. 
Maybe one could be named differently to make the code a little easier to read?

Reply:
I've updated the names of a few variables to better reflect their purpose. I definitely went overboard with `flag`/`flags` 😅